src/components/Transaction.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <div>
+        <table class="table table-striped table-hover table-bordered " v-if="items.length">
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Number of Items</th>
+                    <th>Amount</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="item in items" :key="item.name">
+                    <td>{{ item.item.name }}</td>
+                    <td>
+                        <span v-if="!item.editing" @dblclick="toggleEdit(item)">{{ item.numberOfItems }}</span>
+                        <input v-if="item.editing" @blur="toggleEdit(item)" type="number" v-model="item.numberOfItems">
+                    </td>
+                    <td>{{ item.numberOfItems * item.item.price }}</td>
+                    <td><a class="fa fa-times" @click="removeItem(item)">X</a></td>
+                </tr>
+            </tbody>
+        </table>
+        <p v-if="!items.length">No items have been added.</p>
```

Comment:
Add a "Make Invoice" button that will insert the items in the `Invoice` model, lets just make an invoice in the first cut, worry about payments later